stepconf/stepconf.go

```diff
@@ -141,6 +184,12 @@ func Parse(conf interface{}) error {
 	return nil
 }
 
+// Parse populates a struct with the retrieved values from environment variables
+// described by struct tags and applies the defined validations.
+func Parse(conf interface{}) error {
+	return NewDefaultEnvParser().Parse(conf)
```

Comment:
Maybe a lazy field or just a hardcoded field would be better than creating new "instance" on every call